libs/blocks/merch-card/merch-card.js

```diff
@@ -340,6 +340,17 @@ const setMiniCompareOfferSlot = (merchCard, offers) => {
   merchCard.appendChild(miniCompareOffers);
 };
 
+const updateBigPrices = (merchCard) => {
+  const prices = merchCard.querySelectorAll('strong > em > span[is="inline-price"]');
+
+  prices.forEach((span) => {
```

Comment:
cannot we do this by adding a style to merch-card instead of a class to each prices?
in heavy pages with many merch-cards, this can have a performance hit.

Reply:
The problem is we use a different size in mobile/tablet, that's why the class works best. I can do in JS way but say a device changing screen orientation might not see the expected change in font sizes as per the requirement.

Reply:
@Axelcureno the style can be responsive, I still think this can be done without adding classes to each price.

Reply:
Done